packages/forma-36-react-components/src/components/Tag/Tag.css

```diff
@@ -51,3 +51,10 @@
   color: var(--color-white);
   background-color: var(--color-primary);
 }
+
+.Tag--small {
+  padding: 3px var(--spacing-2xs);
+  font-size: 0.625rem;
+  line-height: 0.625rem;
```

Comment:
tokens?

Reply:
@gui-santos we don't have this token I think - 10px. I guess it might be custom, but if it's needed in other places I could add it to tokens, of course.